### PR TITLE
👔(service):added operations to link API with DB

### PR DESCRIPTION
**Pull request by Tabby709**

### CODE REVIEW

File: libs/api/student-profiles/service/feature/src/lib/models/studentProfile.model.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    email: string;
+    phoneNum: string;
+    dateOfBirth: string;
+    nameOfDegree: string;
+    bio: string;
+    tags: string[];
+    preferredLocation: string;
+    employmentStatus: string;
+    notableAchievements: string;
+    links: string;
```
Comment (Xander-FP) [inline libs/api/student-profiles/service/feature/src/lib/models/studentProfile.model.ts:22]: Links should be a string[].

**Review by Xander-FP — DISMISSED** [review]
Code looks good, just make that one change to the links field and you're good to go.

File: libs/api/student-profiles/service/feature/src/lib/queries/handlers/get-student-profile.handler.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
+import { IQueryHandler, QueryHandler } from "@nestjs/cqrs";
+import { StudentProfilesRepository } from "@graduates/api/student-profiles/repository/data-access";
+import { GetStudentProfileBio, GetStudentProfileDegreeName, GetStudentProfileDOB, GetStudentProfileEmails, GetStudentProfileEmploymentStatus, GetStudentProfileFiles, GetStudentProfileLocation, GetStudentProfileName, GetStudentProfilePFP, GetStudentProfilePhoneNumber, GetStudentProfileSocialMedia, GetStudentProfileTags } from "../impl";
+
+@QueryHandler(GetStudentProfileName)
+export class GetStudentProfileNameHandler implements IQueryHandler<GetStudentProfileName> {
+    constructor(private readonly repository: StudentProfilesRepository) {}
+
+    async execute(query: GetStudentProfileName): Promise<any> {
```
Comment (DanieldeNobrega) [inline libs/api/student-profiles/service/feature/src/lib/queries/handlers/get-student-profile.handler.ts:9]: "Unexpected any. Specify a different type"
I see you're getting this warning on line 9,19 and 29 of this file. 
Can you please check this for me.

**Review by DanieldeNobrega — APPROVED** [review]
Everything looks good, you're good to go.

**Pull request merged**